.github/workflows/cmx-complete.yaml

```diff
@@ -0,0 +1,117 @@
+        cluster: [ {distribution: kind, version: v1.25.3}, {distribution: k3s, version: v1.26} ]
+
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Package Helm Chart for Replicated
+        id: package-helm-chart
+        run: |
+          helm package applications/mlflow/charts/mlflow -u --version ${{ github.event.inputs.version }}
+
+      - name: Create Release
+        id: create-release
+        uses: replicatedhq/replicated-actions/create-release@v1
+        with:
+          api-token: ${{ secrets.REPLICATED_API_TOKEN }}
+          app-slug: ${{ secrets.REPLICATED_APP }}
+          chart: ${{env.chart_name}}-${{ github.event.inputs.version }}.tgz
+
+      - name: Create Customer
+        id: create-customer
+        uses: replicatedhq/replicated-actions/create-customer@v1
+        with:
+          api-token: ${{ secrets.REPLICATED_API_TOKEN }}
+          app-slug: replicated-actions-ci # from the Replicated QA team
+          customer-name: "${{env.chart_name}} CI Test Customer - ${{ github.ref_name }} - ${{ github.sha }}"
+          customer-email: ${{ github.sha }}@example.com
+          license-type: test
+      
+      - name: Create CMX Cluster
+        id: create-cluster
+        uses: replicatedhq/replicated-actions/create-cluster@v1
+        with:
+          api-token: ${{ secrets.REPLICATED_API_TOKEN }}
+          kubernetes-distribution: ${{ matrix.cluster.distribution }}
+          kubernetes-version: ${{ matrix.cluster.version }}
+          timeout-minutes: 5
+      
+      - uses: azure/k8s-set-context@v4
+        with:
+          method: kubeconfig
+          kubeconfig: ${{ steps.create-cluster.outputs.cluster-kubeconfig }}
+
+      - name: Install Dependency Helm Charts
+        id: install-dependency-charts
+        run: |
+          helm repo add cnpg https://cloudnative-pg.github.io/charts
+          helm repo add minio-operator https://operator.min.io
+          helm upgrade --install cnpg \
+            --namespace cnpg-system \
+            --create-namespace \
+            cnpg/cloudnative-pg
+          helm upgrade --install operator \
+              --namespace minio-operator \
+              --create-namespace \
+              minio-operator/operator
+          
+      - name: Create Compatibility Testing
+        id: helm-install-chart
+        uses: replicatedhq/replicated-actions/helm-install@v1
+        with:
+          kubeconfig: ${{ steps.create-cluster.outputs.cluster-kubeconfig }}
+          helm-path: helm
+          registry-username: ${{ github.ref_name }}@example.com
+          registry-password: ${{ steps.create-customer.outputs.license-id }}
+          chart: oci://registry.replicated.com/${{ secrets.REPLICATED_APP }}/release__${{ steps.create-release.outputs.release-sequence }}/${{env.chart_name}}
+          name: ${{env.chart_name}}
+          namespace: test-${{env.chart_name}}
+          run-preflights: false
+
+      - name: Run Compatibility Testing
```

Comment:
Should we just do a `--wait` in the step above to avoid this? And then perhaps this stage can be a check against a `/healthz` endpoint or something?

Reply:
Good idea, I have add a wait for helm to be ready action. For the `/healthz` endpoint. Do we have one in mflow?

Reply:
I found the health endpoint. Now the example will check the application /health